src/DotNet.Testcontainers.Tests/Fixtures/OracleDbFixture.cs

```diff
@@ -17,7 +17,7 @@ public class OracleDbFixture : IAsyncLifetime
     public OracleDbFixture()
```

Comment:
You need to rebase on top of `develop` first. I did some clean up. You had typos and inconsistent class / file names in your PR.